protocols/gossipsub/src/behaviour.rs

```diff
@@ -3684,7 +3684,8 @@ impl<C: DataTransform, F: TopicSubscriptionFilter> fmt::Debug for Behaviour<C, F
             .field("fanout", &self.fanout)
             .field("fanout_last_pub", &self.fanout_last_pub)
             .field("mcache", &self.mcache)
-            .field("heartbeat", &self.heartbeat)
+            // This doesn't derive `Debug` trait yet
+            // .field("heartbeat", &self.heartbeat)
```

Comment:
Just to make sure this does not go unnoticed before merging.

Reply:
https://github.com/antifuchs/futures-ticker/pull/11

Reply:
Thanks to both of you!

Reply:
fixed with https://github.com/libp2p/rust-libp2p/pull/3973/commits/9a00bab0b712679d2ca0b0a96df28949b2f117c7

Reply:
Thank you @ozkanonur.